zerver/tests/test_subs.py

```diff
@@ -6058,7 +6058,7 @@ def test_gather_subscriptions(self) -> None:
             polonius.id,
         ]
 
-        with self.assert_database_query_count(43):
+        with self.assert_database_query_count(46):
```

Comment:
This is a significant increase; is this because it sends 3 DMs?

Reply:
ya, mainly because of the `Notification Bot` Dm saying you are subscribed to this channel.